### PR TITLE
Changed "Choose a status" label to "Set a status"

### PR DESCRIPTION
**Pull request by nicorsm**
## What's new in this PR?

Changed "Choose a status" label for Availability ActionSheet to "Set a status"

### CODE REVIEW

**Comment by nicorsm** [comment]
Should I remove the old translation entry for "Choose a status"?

**Comment by mikeger** [comment]
@nicorsm if the string is not used it makes sense to remove it to save some work for the translators.

File: Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift [hunk 1]
```diff
@@ -103,7 +103,7 @@ extension AvailabilityTitleView {
     
     var actionSheet: UIAlertController {
         get {
-            let alert = UIAlertController(title: "availability.message.title".localized, message: nil, preferredStyle: .actionSheet)
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift]: So `availability.message.title` is unused now? We should delete it then.

Comment (nicorsm) [reply Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleView.swift]: Exactly

**Pull request merged**